internal/nvmeof/tests/Dockerfile.nvmeof-test

```diff
@@ -0,0 +1,45 @@
+# Dockerfile.nvmeof-test
```

Comment:
Complex testing like this should be in a separate commit (of the same PR). Simple unit-tests for utility functions can be in the same commit as where the function is introduced. Splitting large commits into smaller chunks makes it easier to review.